enterprise/internal/scim/user.go

```diff
@@ -70,38 +76,75 @@ func (h *UserResourceHandler) Get(r *http.Request, idStr string) (scim.Resource,
 // An empty list of resources will be represented as `null` in the JSON response if `nil` is assigned to the
 // Page.Resources. Otherwise, is an empty slice is assigned, an empty list will be represented as `[]`.
```

Comment:
```suggestion
// Page.Resources. Otherwise, if an empty slice is assigned, an empty list will be represented as `[]`.
```

Not really part of the PR but saw it :P